src/cp/sliprules.cxx

```diff
@@ -0,0 +1,189 @@
+#include "sliprules.h"
+
+namespace neml {
+
+double SlipRule::sum_slip(const Symmetric & stress, const Orientation & Q, 
```

Comment:
waht's the plan for including self and latent hardening coefficients? A new class?

Reply:
Yes, it will be a new SlipRule.